client/scss/components/_list_card.scss

```diff
@@ -0,0 +1,105 @@
+//* @define list-card
+
+$list-card-number-size: 96px;
+
+.list-card {
```

Comment:
> What do we mean by card?

In this instance, I think we're calling it a card is solely because it looks kinda card-y.